vital/algo/estimate_fundamental_matrix.h

```diff
@@ -12,66 +12,84 @@
 
 #include <vital/vital_config.h>
 
-#include <vector>
 #include <memory>
+#include <vector>
 
 #include <vital/algo/algorithm.h>
 #include <vital/types/feature_set.h>
-#include <vital/types/match_set.h>
 #include <vital/types/fundamental_matrix.h>
+#include <vital/types/match_set.h>
 
 namespace kwiver {
+
 namespace vital {
+
 namespace algo {
 
-/// An abstract base class for estimating a fundamental matrix from matching 2D points
+/// An abstract base class for estimating a fundamental matrix from matching 2D
+/// points
 class VITAL_ALGO_EXPORT estimate_fundamental_matrix
-  : public kwiver::vital::algorithm_def<estimate_fundamental_matrix>
+  : public kwiver::vital::algorithm_def< estimate_fundamental_matrix >
 {
 public:
+
   /// Return the name of this algorithm
-  static std::string static_type_name() { return "estimate_fundamental_matrix"; }
+  static std::string
+  static_type_name()
+  {
+    return "estimate_fundamental_matrix";
+  }
 
   /// Estimate an fundamental matrix from corresponding features
+
   /**
    * \param [in]  feat1 the set of all features from the first image
    * \param [in]  feat2 the set of all features from the second image
-   * \param [in]  matches the set of correspondences between \a feat1 and \a feat2
+   * \param [in]  matches the set of correspondences between \a feat1 and \a
+   * feat2
```

Comment:
Prior comment about not fixing Doxygen stuff right now, I'm not actually sure if Doxygen like this...
```suggestion
   * \param [in]  matches the set of correspondences between \a feat1 and
   *                      \a feat2
```

Reply:
I tried both formats and Doxygen doesn't care.  They both look the same and look good.  I agree however, that your suggestion is an improvement when looking at the source.